packages/react/widget/src/hooks/use-portal-widgets.ts

```diff
@@ -0,0 +1,59 @@
+import { useMemo, useRef, useState } from 'react';
+import { createElement } from '../util/element';
+import { widgetRender } from '../render/render';
+import { IWidgetModuleProvider, WidgetProps } from '@equinor/fusion-framework-module-widget';
+import { Fusion } from '@equinor/fusion-framework';
+
+export const usePortalWidgets = <TProps extends WidgetProps>(
```

Comment:
this indicates plural, but only returns one
missing description of what this hook does,
please give example usage.

when do you use this hook outside `BaseWidget`, cant it just been memo inside there?

Reply:
This is an internal hook not used by anything other hhan the BaseWidget.

naming wise i thin useWidget or useWidgetLoad is better. Should i change?

Do you whant exaple docs for internal code?

Reply:
ill cahnge it to useLoadWidget